src/main/java/seedu/address/model/BorrowerRecords.java

```diff
@@ -91,6 +91,35 @@ public Borrower getBorrowerFromId(BorrowerId id) throws NullPointerException {
         return borrowersMap.get(id);
     }
 
+    /**
+     * Returns true if the list contains an equivalent borrower as the given argument.
+     */
+    public boolean contains(Borrower toCheck) {
+        requireNonNull(toCheck);
+        return listOfBorrowers.stream().anyMatch(toCheck::equals);
+    }
+    // why does hasBorrower check by phone number or email only????
```

Comment:
I think `hasBorrower()` might be used to check when a new borrower is registering, but everything else uses `equals()`. Need to confirm with @linyutinglyt

Reply:
Yup it was used to check when a new borrower is register!! Initially it was checking name, phone number and email but then we realise another borrower using a different name and email will be able to register which should not be the case.